deployer/tf_plugin_client.go

```diff
@@ -79,6 +90,7 @@ func NewTFPluginClient(
 	keyType string,
 	network string,
 	substrateURL string,
+	passedRelayURL string,
 	passedRmbProxyURL string,
```

Comment:
I don't think we need this parameter anymore for proxying, but do we need it for grid rest API?